examples/Makefile-cc2538

```diff
@@ -39,7 +39,7 @@ NM                              = arm-none-eabi-nm
 RANLIB                          = arm-none-eabi-ranlib
 OBJCOPY                         = arm-none-eabi-objcopy
 
-BuildJobs                      ?= 99
+BuildJobs                      ?= 10
```

Comment:
Why 10?  Do you know the root cause?  Seems like your VM has a big security hole if user processes can cause your machine to reboot.